site/diagrams/sequenceDiagrams/assertDeviceRegistration_Success.txt

```diff
@@ -0,0 +1,12 @@
+title Assert Device Registration (success)
+
+Client->+Device Registration Service: transfer("registration/tenant", "assert", device-id)
+note over Device Registration Service:
+lookup device and check enabled flag
+generate assertion
+end note
+Device Registration Service->Client: transfer("registration/tenant/${reply-to}", device-id, assertion)
```

Comment:
maybe we should also include the status code (200) in this case? we do include it in all other interactions.

Reply:
Thought about this already myself, so this sounds like a good point!
Will include it here and the Credentials API diagrams, too.